code/modules/mining/mine_turfs.dm

```diff
@@ -338,7 +338,7 @@
 	new turf_type(T)
 /turf/open/floor/plating/asteroid/airless/cave/proc/SpawnMonster(turf/T)
 	if(prob(30))
-		if(istype(loc, /area/mine/explored))
+		if(istype(loc, /area/mine/explored) || istype(loc, /area/lavaland/surface/outdoors/labor_camp))
```

Comment:
Or make a new area lavaland/explored and use that instead of the gulag area since it'd make sense to reuse that around the base as well